code/__DEFINES/traits/declarations.dm

```diff
@@ -515,6 +515,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_RIGHT_EYE_SCAR "right_eye_scar"
 #define TRAIT_LEFT_EYE_SCAR "left_eye_scar"
 
+/// Mob doesn't play body drop sfx when dying/in crit
+#define TRAIT_NO_BODY_DROP_SFX "NO_BODY_DROP_SFX"
```

Comment:
this doesn't make any sense to me it's applied on init. also the motivation doesn't seem tied to it at all